api-auto-registration/installation.hbs.md

```diff
@@ -0,0 +1,78 @@
+# Install API Auto Registration
+
+This document describes how to install API Auto Registration from the Tanzu Application Platform package repository.
+
+>**Note:** Use the instructions on this page if you do not want to use the full or run profile to install packages.
+Only the full and run profiles includes API Auto Registsration.
+For more information about profiles, see [About Tanzu Application Platform components and profiles](../about-package-profiles.md).
+
+## <a id='parameters'></a>Parameters
+API Auto Registration uses the following parameters in `tap-values.yaml`
+
+```yaml
+api_autoregistration:
+  tap_gui_url: <URL TO TAP GUI>
+  cluster_name: <CLUSTER NAME AS SHOWN IN LIFECYCLE>
+```
+
+>**Note:** The value of `cluster_name` is what you will see in the `lifecycle` property of your API Entitites 
+
+If you are installing with full profile you will get the following default values, but keep in mind you can override them if need be.
+
+```yaml
+api_autoregistration:
+  tap_gui_url: http://server.tap-gui.svc.cluster.local:7000 
+  cluster_name: <Value from shared.ingressDomain or 'dev' if not found>
+```
+
+When using a mutli-cluster installtion using the run cluster profile or without a profile you will need to set the `api_autoregistration` parameters manually.
+
+You can locate the `tap_gui_url` by going to the cluster that is running TAP GUI and executing the following commands
+
+```console
+kubectl get secret tap-values -n tap-install -o jsonpath="{.data['tap-values\.yaml']}" | base64 -d | yq '.tap_gui.app_config.backend.baseUrl'  
+``` 
+
+## <a id='prereqs'></a>Prerequisites
```

Comment:
This is not a focused comment but this Prerequisites section seems to kinda break the flow? I feel like it should go before the Parameters section and remove/move the "Located the TAP GUI url using the instructions mentioned above" sentence out?

Reply:
Rearranged and removed the suggested sentence